Makefile

```diff
@@ -23,7 +23,7 @@ format:
 
 .PHONY: lint
 lint:
-	flake8 pydantic/ tests/
+	flake8 pydantic/ tests/ docs/examples/
```

Comment:
I think we should move this into the normal docs build, `make docs`.

Also we should adjust the line limit to 80 or whatever when linting docs.

That will avoid your issues with linting only on 3.8.